<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InLink</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="JobsTabBtn">
        <property name="text">
         <string>Jobs</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/job.png</normaloff>:/job.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tabsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="MyNetworkTabBtn">
        <property name="text">
         <string>My Network</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/cloud.png</normaloff>:/cloud.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tabsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="messaginTabBtn">
        <property name="text">
         <string>Messaging</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/comment.png</normaloff>:/comment.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tabsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="homeTabBtn">
        <property name="styleSheet">
         <string notr="true">#homeTabBtn {
    qproperty-iconSize: 24px;
}
#homeTabBtn::icon {
    position: absolute;
    top: 0;
    left: 50%;
    transform: translateX(-50%);
}
#homeTabBtn::text {
    position: absolute;
    bottom: 0;
    left: 50%;
    transform: translateX(-50%);
}</string>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/home.png</normaloff>:/home.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tabsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="SearchBoxTabButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="MeTabBtn">
        <property name="text">
         <string>Me</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/user.png</normaloff>:/user.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">tabsButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QScrollArea" name="Area">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>774</width>
           <height>486</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="tabsButtonGroup"/>
 </buttongroups>
</ui>
